### ISSUE TITLE
xmrig-cuda didn't fix kapow problem

### ISSUE THREAD

**Issue by picklejohndoe** [UnamSanctam/UnamWebPanel#394]
hi so I have a problem using Nvidia GPU and I saw you said to use xmrig-cuda. I downloaded the newest one then I downloaded v10 both times it didn't work. in running it on a GeForce gt 710, maybe its the vram that can't run it, but since there's that problem, would it be possible to make it try running a sequence until its running a miner?

is there a way to pair xmrig-cuda and the silent miner to fix the problem with Nvidia graphics card?

also is running GPU miner and a CPU miner at the same time possible with the new silent miner builder?

also I was thinking, similar to the way PGP works is it possible if you make a encryption file builder that will generate different types of encryptions to bypass AV and even though its open source by making where there's a new type of encryption months and possible years from now it can still be encrypted and undetectable?

**Comment by UnamSanctam** [comment]
>hi so I have a problem using Nvidia GPU and I saw you said to use xmrig-cuda. I downloaded the newest one then I downloaded v10 both times it didn't work. in running it on a GeForce gt 710, maybe its the vram that can't run it, but since there's that problem, would it be possible to make it try running a sequence until its running a miner?

Yes your GPU does not have enough VRAM to mine Ravencoin if you're attempting that, a GeForce GT 710 can't probably mine anything, though possibly Neurai if nothing else is using any VRAM. But mining KawPow with the current miner without xmrig-cuda works just fine (usually any problems reported in issues aren't actually due to the miner but instead due to something with the computer, GPU or network of the user).

>is there a way to pair xmrig-cuda and the silent miner to fix the problem with Nvidia graphics card?

Xmrig is even worse at mining KawPow than the current GPU miner in the silent miner, but technically you can do so, though it's complicated to do.

>also is running GPU miner and a CPU miner at the same time possible with the new silent miner builder?

Yes you can just add both a CPU miner and GPU miner and it will work.

>also I was thinking, similar to the way PGP works is it possible if you make a encryption file builder that will generate different types of encryptions to bypass AV and even though its open source by making where there's a new type of encryption months and possible years from now it can still be encrypted and undetectable?

Each build uses different encryption keys already, the encryption is only a small part of detections.

**Comment by picklejohndoe** [comment]
so is there a way to have the miner run certain ones for gpu? so like if there's someone else that has the same gpu as me it would ring X but if it better and meets the gpu criteria for KawPow then it would mine KawPow algorithm

**Comment by UnamSanctam** [comment]
RandomX is a CPU algorithm (for example Monero uses it) and KawPow is a GPU algorithm, your issue is in regards to VRAM usage, in which case what matters is really what coin you mine. So for example Neurai uses 1.7GB of VRAM, which is very little compared to many coins..
